apps/modernization-ui/src/apps/patient/profile/entry/EntryModal.tsx

```diff
@@ -27,7 +26,7 @@ export const EntryModal = ({ modal, id, title, children, overflow = false, class
             ref={modal}
             className={classNames(styles.modal, className, { [styles.overflow]: overflow })}>
             <header>
-                <Heading level={2}>{title}</Heading>
+                <h2 className="heading_heading__Xr81l heading_two__eM+ke margin-0">{title}</h2>
```

Comment:
issue:  The classes `heading_heading__Xr81l` and  `heading_two__eM+ke` are auto generated classes from SASS.  They are not guaranteed to exist.